package.json

```diff
@@ -63,6 +63,7 @@
     "sinon": "^2.2.0",
     "sw-testing-helpers": "^1.0.2",
     "tmp": "0.0.31",
-    "uppercamelcase": "^3.0.0"
+    "uppercamelcase": "^3.0.0",
+    "webpack": "^2.6.0"
```

Comment:
No top level is fine here (and preferable here).

the *-build and *-cli have tests to check dependencies and it's much easier if devDep's are in top level and only deps are in the package itself.